src/components/ContactForm/index.jsx

```diff
@@ -7,11 +7,12 @@ import firebase from "../../firebaseConfig";
 import { getErrorClass } from "../../utils/formErrorHelpers";
 import "./index.scss";
 
-export default function ContactForm() {
+export default function ContactForm({ handleShow }) {
   const { register, handleSubmit, errors, reset } = useForm();
   const { t } = useTranslation();
 
   const onSubmit = (data) => {
+    handleShow();
```

Comment:
I think you should first check if the message has been saved successfully and then eaither say that "thank you ...." or "Oops! something happend try again ...". you can use catch for error I think

Reply:
it won't come to submit stage, so I think we don't need it 
<img width="1435" alt="Screen Shot 2020-11-18 at 22 19 27" src="https://user-images.githubusercontent.com/67760504/99577312-36012f00-29ec-11eb-8edb-544ebb767a2d.png">

Reply:
and if it's sent to firebase, then it will be saved